tests/e2e/specs/Pools.spec.js

```diff
@@ -15,6 +15,25 @@ import {
 
 import { nodeTypes } from '../support/constants';
 
+function boundaryEventIsCloseToTask() {
```

Comment:
What do you think about renaming this to something like `assertBoundaryEventIsCloseToTask`? Similar to what I've done with the `assertDownloadedXmlContainsExpected` helpers. Currently, for me, this reads like it should return a boolean.

Reply:
Sure. I thought about making it share-able. But it relies on there only being one task with a boundary event.